pyad/adcontainer.py

```diff
@@ -77,7 +77,11 @@ def create_computer(self, name, enable=True,optional_attributes={}):
         """Create a new computer object in the container"""
         try:
             obj = self.__create_object('computer', name)
-            obj.Put('sAMAccountName', name)
+            obj.Put('sAMAccountName', name + '$')
+            if enable:
+                obj.Put('userAccountControl', 4128)
+            else:
+                obj.Put('userAccountControl', 4130)
             obj.SetInfo()
             pyadobj = ADComputer.from_com_object(obj)
             if enable:
```

Comment:
Seems like this needs to change if we're changing how enable works above. is there any rason that those values shoouldn't just be set in`.enable()`?

Reply:
It has to be set at creation otherwise you'll end with a user account instead of a computer one (see #100).

Reply:
Makes sense. DO we still need to call `.enable()`?

Reply:
Yes, by default the object is disabled.